test_scripts/API/VehicleData/Refactor_Fuel_Information/004_OnVehicleData_with_new_fuelRange_structure.lua

```diff
@@ -0,0 +1,30 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
Referring to this comment https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2413#discussion_r451132043, I think this parameter should be covered in the `OnVehicleData/001_Success_flow.lua` test and this new test should be removed